package.json

```diff
@@ -52,9 +53,12 @@
     "webpack-dev-server": "^1.16.2",
     "webpack-validator": "^2.2.9"
   },
-  "dependencies": {
+  "peerDependencies": {
     "d3": "^4.3.0",
-    "react": "^0.14.7"
+    "react": "*"
```

Comment:
it's probably not the case that the lib can be this flexible.  Not sure if there would be a bottom limit to compatability.  Given that React continues to deprecate things, though, there's definitely an upper limit.